src/main/java/org/eclipse/tractusx/puris/backend/common/api/domain/repository/ResponseRepository.java

```diff
@@ -21,14 +21,17 @@
 package org.eclipse.tractusx.puris.backend.common.api.domain.repository;
 
 import org.eclipse.tractusx.puris.backend.common.api.domain.model.Request;
+import org.eclipse.tractusx.puris.backend.common.api.domain.model.Response;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
 
 import java.util.UUID;
 
 /**
  * Repository to access Responses
  */
-public interface ResponseRepository extends JpaRepository<Request, UUID> {
+@Repository
+public interface ResponseRepository extends JpaRepository<Response, UUID> {
```

Comment:
Do we still need this Repository? This is the ResponseRepository that returns a Request.

Reply:
I think in general it seems necessary to have the possibility to look up which responses were sent out from your side to external business partners. Another question is however if it is necessary to store it via JDBC/SQL. A possible alternative might be to store the data to a log file.

Reply:
Just do a logging on info level like "response to request id {request id} for following materials: {material number customer,  material number supplier, material number cx}; {next entry}"

We'll add an issue to may add it in future.